appmonitoring/ts/src/server.ts

```diff
@@ -1,13 +1,24 @@
-﻿import * as fs from "fs";
-import * as https from "https";
+﻿import * as https from "https";
 import { ContentProcessor } from "./ContentProcessor.js";
 import { logger, Metrics } from "./LoggerWrapper.js";
 import { AppMonitoringConfigCR, IRootObject } from "./RequestDefinition.js";
 import { K8sWatcher } from "./K8sWatcher.js";
 import { AppMonitoringConfigCRsCollection } from "./AppMonitoringConfigCRsCollection.js"
+import fs from "fs";
+import { CertificateManager } from "./CertificateGenerator.js";
 
+const containerMode = process.env.CONTAINER_MODE;
+
+if ("secrets-manager".localeCompare(containerMode) === 0) {
+
```

Comment:
Empty line?

Reply:
without this I have no proof I actually wrote this code :(